src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -799,148 +884,236 @@ export const NodeSelectors: React.FC<NodeSelectors> = ({
+                                marginRight: '0',
+                                textAlign: 'left',
+                                direction: 'rtl',
+                                color: 'var(--N000)',
+                            }),
+                            input: (base, state) => ({ ...base, caretColor: 'var(--N000)', color: 'var(--N000)' }),
+                            option: (base, state) => ({
+                                ...base,
+                                backgroundColor: state.isFocused ? 'var(--N100)' : 'white',
+                                color: 'var(--N900)',
+                                textOverflow: 'ellipsis',
+                                overflow: 'hidden',
+                                whiteSpace: 'nowrap',
+                                direction: 'rtl',
+                            }),
+                        }}
+                    />
+                </div>
             </div>
-        </div>
-        {Array.isArray(allContainers) && (params.tab === NodeDetailTabs.LOGS || params.tab === NodeDetailTabs.TERMINAL) && (
-            <>
-                <span style={{ width: '1px', height: '16px', background: '#0b0f22' }} />
-                <div className="events-logs__dropdown-selector">
-                    <span className="events-logs__label">Containers</span>
-                    <div style={{ width: '175px' }}>
-                        <Select placeholder="Select Container"
-                            options={allContainers[0] && allContainers[0].map((container) => ({ label: container, value: container }))}
-                            value={containerName ? { label: containerName, value: containerName } : null}
+            {Array.isArray(allContainers) &&
+                (params.tab === NodeDetailTabs.LOGS || params.tab === NodeDetailTabs.TERMINAL) && (
+                    <>
+                        <span style={{ width: '1px', height: '16px', background: '#0b0f22' }} />
+                        <div className="events-logs__dropdown-selector">
+                            <span className="events-logs__label">Containers</span>
+                            <div style={{ width: '175px' }}>
+                                <Select
+                                    placeholder="Select Container"
+                                    options={
+                                        allContainers[0] &&
+                                        allContainers[0].map((container) => ({ label: container, value: container }))
+                                    }
+                                    value={containerName ? { label: containerName, value: containerName } : null}
+                                    onChange={(selected) => {
+                                        selectContainer((selected as any).value)
+                                        onLogsCleared()
+                                    }}
+                                    styles={{
```

Comment:
same here

Reply:
It is there in multiple place can we skip and do it all over together?